api/solarperformanceinsight_api/models.py

```diff
@@ -285,23 +358,50 @@ class Inverter(BaseModel):
 class PVSystem(BaseModel):
     """Parameters for an entire PV system at some location"""
 
-    name: userstring = Field(..., description="Name of the system")
-    latitude: confloat(ge=-90, le=90) = Field(
-        ..., description="Latitude of the system in degrees North"
+    name: str = UserString(
+        ...,
+        description="Name of the system",
     )
-    longitude: confloat(ge=-180, le=180) = Field(
-        ..., description="Longitude of the system in degrees East"
+    latitude: float = Field(
+        ..., description="Latitude of the system in degrees North", ge=-90, le=90
     )
-    elevation: confloat(ge=-300) = Field(
-        ..., description="Elevation of the system in meters"
+    longitude: float = Field(
+        ..., description="Longitude of the system in degrees East", ge=-180, le=180
     )
-    albedo: confloat(ge=0) = Field(
-        ..., description="Albedo of the surface around the system"
+    elevation: float = Field(
+        ..., description="Elevation of the system in meters", ge=-300
+    )
+    albedo: float = Field(
+        ..., description="Albedo of the surface around the system", ge=0
     )
     inverters: List[Inverter] = Field(
         ..., description="Inverters that are connected to make up this system"
     )
 
+    class Config:
+        schema_extra = {"example": SYSTEM_EXAMPLE}
+
 
-class StoredPVSystem(PVSystem):
+class PVSystemID(BaseModel):
     system_id: UUID = Field(..., description="Unique identifier of the system")
+
+    class Config:
+        schema_extra = {
+            "example": {"system_id": "6b61d9ac-2e89-11eb-be2a-4dc7a6bcd0d9"}
+        }
+
+
+class StoredPVSystem(PVSystemID):
+    created_at: dt.datetime = Field(..., description="Datetime system was created")
+    modified_at: dt.datetime = Field(..., description="Datetime system last modified")
+    definition: PVSystem
```

Comment:
@lboeman This change will definitely affect the dashboard. I think it makes sense to separate the pvsystem definition from storage metadata (system_id, created_at, modified_at, maybe others in the future). I think this will make life easier when we need to process the data i.e. just pass storedpvsystem.definition vs the whole thing with the extra fields. What do you think?

Reply:
I think this is the right way to go. 
One thing that was kind of a pain point on the arbiter dashboard was different uuid field names. Particularly when trying to write generic render code for similar objects with minor tweaks. It might make things a little easier if we had a pattern like `{'type': 'system', 'id': <uuid>}` over a field name like `system_id`, which seems to follow the pattern of separation of the storage metadata and object specification.
But since we're not storing too much here that might not be such an issue and I'm not opposed to just continuing with this pattern as this is cleaner on the python side. Just something to consider going forward.

Reply:
> It might make things a little easier if we had a pattern like {'type': 'system', 'id': <uuid>} over a field name like system_id, which seems to follow the pattern of separation of the storage metadata and object specification.

sounds good to me

Reply:
actually decided to use `object_id` and `object_type` to avoid conflicting w/ the builtin python functions